### ISSUE TITLE
Graphics

### ISSUE THREAD

**Issue by nacmo2807** [doublespeakgames/adarkroom#643]
For an additional immersive gaming experience, to avoid disengagement between game and player, there should be contrasting graphics to maintain focus.

**Comment by j-f1** [comment]
A lack of graphics is one of A Dark Room’s differentiating features.

**Comment by nacmo2807** [comment]
What about adding icons, for fire, woods, meat, etc?

**Comment by Continuities** [comment]
Nope. ADR is, and always will be, text-only.

**Comment by Jaredcscott** [comment]
I agree with @Continuities and @j-f1. The way that ADR appeared to be so simple, yet expands into such an immersive experience is what hooked me into the game.

**Comment by JordMonte512** [comment]
I think the simplicity of the game is what makes it different yet so engaging.

My question is along the lines of following the Mobile App. Could the background of the page in light mode change from bright to a dark grey as the fire starts going out, to draw the player to the stoke fire button. I think adding this in a very subtle but noticeable way would make the player feel like they are in the room, watching the fire go out.

I hope that makes sense, I couldn't think how to word it.

**Comment by esaruoho** [comment]
@nacmo2807 i think this conversation has been had - and the ticket can be closed.
